src/rpc/net.cpp

```diff
@@ -407,7 +407,7 @@ static UniValue getnettotals(const JSONRPCRequest& request)
                    {
                        {RPCResult::Type::NUM, "totalbytesrecv", "Total bytes received"},
                        {RPCResult::Type::NUM, "totalbytessent", "Total bytes sent"},
-                       {RPCResult::Type::NUM_TIME, "timemillis", "Current UNIX time in milliseconds"},
+                       {RPCResult::Type::NUM_TIME, "timemillis", UNIX_EPOCH_TIME + " in milliseconds"},
```

Comment:
nit: Any reason to remove "Current "?

Reply:
> nit: Any reason to remove "Current "?

Without any other qualifier I thought it was self-understood but don't mind adding back "Current" if reviewers think it's unclear.

Reply:
Re-added while retouching to fix the extra line.